scipy/special/tests/test_support_alternative_backends.py

```diff
@@ -89,4 +89,4 @@ def test_support_alternative_backends(xp, f_name_n_args, dtype, shapes):
     ref = xp.asarray(f(*args_np), dtype=dtype_xp)
 
     eps = np.finfo(dtype_np).eps
-    xp_assert_close(res, ref, atol=10*eps)
+    xp_assert_close(res, ref, atol=10*eps, check_0d=False)
```

Comment:
Should `special` not be using the `_no_0D` version too? From a quick browse of the docs, most return values are documented to be of type `scalar or ndarray`.